text-match.cc:475:8: error: no viable conversion from 'pan::StringView' to 'std::string'

Index: pan/general/string-view.h
--- pan/general/string-view.h.orig
+++ pan/general/string-view.h
@@ -95,7 +95,7 @@ namespace pan
          std::string to_string () const {return empty()
                                          ? std::string()
                                          : std::string(str,str+len); }
-         operator const std::string () const { return to_string(); }
+         operator std::string () const { return to_string(); }
 
          StringView substr (const char * start, const char * end) const;
          void substr (const char * start, const char * end, StringView& setme) const;
